kitsune/sumo/static/sumo/js/form-wizard-sign-in-step.js

```diff
@@ -20,7 +20,7 @@ export class SignInStep extends BaseFormStep {
           </p>
 
           <p class="for-sign-in">
-            ${gettext("A Firefox account allows you to securely sync your data when signing in from other devices and gives access to other fantastic Mozilla services.")}
+            ${gettext("A Firefox account allows you to securely sync your data when logging in from other devices and gives access to other fantastic Mozilla services.")}
```

Comment:
Hey @hannajones - it turns out that we _did_ want this to be "signing", but the document was never updated to reflect that. I've corrected the document, and then I think we can revert this change.

Reply:
thanks for clarifying! In that case I'll just commit the whitespace removal I guess 😂